### PR TITLE
Logpay | Create payment methods

### PR DESCRIPTION
**Pull request by phylor**
Adds support to create a logpay payment method.

### CODE REVIEW

File: lib/ioki/apis/passenger_api.rb [hunk 1]
```diff
@@ -113,6 +113,20 @@ class PassengerApi
         base_path:   [API_BASE_PATH],
         model_class: Ioki::Model::Passenger::Station
       ),
+      Endpoints::Create.new(
+        :logpay_customer,
+        base_path:            [API_BASE_PATH],
+        path:                 'logpay/customer',
```
Comment (rmehner) [inline lib/ioki/apis/passenger_api.rb:119]: Same path thing as in the previous PR :)

File: lib/ioki/apis/passenger_api.rb [hunk 2]
```diff
@@ -113,6 +113,20 @@ class PassengerApi
...
+        model_class:          Ioki::Model::Passenger::LogpayUrl,
+        outgoing_model_class: Ioki::Model::Passenger::LogpayCustomer
+      ),
+      Endpoints::Create.new(
+        :logpay_payment_method,
+        base_path:            [API_BASE_PATH],
+        path:                 'logpay/payment_method',
```
Comment (rmehner) [inline lib/ioki/apis/passenger_api.rb:126]: See above :)

**Review by rmehner — APPROVED** [review]

File: lib/ioki/apis/passenger_api.rb (hunk 1 above)
Comment (phylor) [reply lib/ioki/apis/passenger_api.rb]: Changed!

File: lib/ioki/apis/passenger_api.rb (hunk 2 above)
Comment (phylor) [reply lib/ioki/apis/passenger_api.rb]: Changed!

File: lib/ioki/apis/passenger_api.rb (hunk 1 above)
Comment (phylor) [reply lib/ioki/apis/passenger_api.rb]: That doesn't work out of the box with `Endpoints::Create`. I'll merge it as is and create another PR for the consistency.

**Pull request merged**